themes/ddbasic/sass/components/ting-object/ting-object.scss

```diff
@@ -712,6 +714,9 @@
         clear: both;
         padding: 28px 0 26px;
         border-top: 1px solid $charcoal-opacity-light;
+        .no-holdings & {
```

Comment:
You have removed the code that added the no-holdings class or is it added by ill module in som other way?

Reply:
It's now being added by the `ding_ill` module. :)

Reply:
👍